fragments/header.sh

```diff
@@ -157,6 +157,10 @@ DIALOG_LIST_ITEM_NAME=""
 # listitem.
 # When the variable is unset, progress will be sent to Swift Dialog's main progress bar.
 
+# Add resulting App to the Dock using dockutil
```

Comment:
Lets call it `Add installed app` rather than resulting app, and describe that the app will be added at the end of the user's current dock. Can we also add a line that dockutil has an Installomator label. Perhaps remove the dockutil GitHub URL and just reference that a dockutil label exits.